xbmc/video/tags/VideoTagLoaderFFmpeg.cpp

```diff
@@ -255,3 +257,27 @@ CInfoScanner::INFO_TYPE CVideoTagLoaderFFmpeg::LoadMP4(CVideoInfoTag& tag,
 
   return hasfull ? CInfoScanner::FULL_NFO : CInfoScanner::TITLE_NFO;
 }
+
+// https://wiki.multimedia.cx/index.php/FFmpeg_Metadata#AVI
+CInfoScanner::INFO_TYPE CVideoTagLoaderFFmpeg::LoadAVI(CVideoInfoTag& tag,
+                                                       std::vector<EmbeddedArt>* art)
+{
+  AVDictionaryEntry* avtag = nullptr;
+  while ((avtag = av_dict_get(m_fctx->metadata, "", avtag, AV_DICT_IGNORE_SUFFIX)))
+  {
+    if (strcmp(avtag->key, "title") == 0)
+      tag.SetTitle(avtag->value);
+    else if (strcmp(avtag->key, "genre") == 0)
+      tag.SetGenre(StringUtils::Split(avtag->value, " / "));
+    else if (strcmp(avtag->key,"date") == 0)
+      tag.SetYear(atoi(avtag->value));
+    else if (strcmp(avtag->key, "track") == 0)
+      tag.m_iTrack = std::stoi(avtag->value);
+    else if (strcmp(avtag->key, "album") == 0)
+      tag.SetAlbum(avtag->value);
+    else if (strcmp(avtag->key, "artist") == 0)
+      tag.SetArtist(StringUtils::Split(avtag->value, " / "));
+  }
+
+  return CInfoScanner::TITLE_NFO;
```

Comment:
for rich info we should return FULL_NFO no ?

Reply:
The discussion for MP4 was that as long as neither `description` nor `synopsis` were present, this meant user wanted to download this info from IMDB. Since AVI does not contains either keys (https://wiki.multimedia.cx/index.php/FFmpeg_Metadata#AVI), I believe the behavior is always INFO (only).

Reply:
okay, but in that case reading additional tags is nop.

Reply:
Indeed! Updated the PR just now.